source/emissions/message/index.rst

```diff
@@ -51,10 +51,54 @@ The MESSAGE model includes a detailed representation of energy-related and - via
 
 CO2 emissions of fossil fuels for the entire energy system are accounted for at the resource extraction level by applying the CO2 emission factors listed in :numref:`tab-emissionfactor` to the extracted fossil fuel quantities. In this economy-wide accounting, carbon emissions captured in CCS processes remove carbon from the balance equation, i.e. they contribute with a negative emission coefficient. In parallel, a sectoral acounting of CO2 emissions is performed which applies the same emission factors to fossil fuels used in individual conversion processes. In addition to conversion processes, also CO2 emissions from energy use in fossil fuel resource extraction are explicitly accounted for. A relevant feature of MESSAGE in this context is that CO2 emissions from the extraction process increase when moving from conventional to unconventional fossil fuel resources (McJeon et al., 2014 :cite:`mcjeon_gas_2014`).
 
-CO2 mitigation options in the energy system include technology and fuel shifts; efficiency improvements; and CCS. A large number of specific mitigation technologies are modeled bottom-up in MESSAGE with a dynamic representation of costs and efficiencies. As mentioend above, MESSAGE also includes a detailed representation of carbon capture and sequestration from both fossil fuel and biomass combustion. 
+CO2 mitigation options in the energy system include technology and fuel shifts; efficiency improvements; and CCS. A large number of specific mitigation technologies are modeled bottom-up in MESSAGE with a dynamic representation of costs and efficiencies. As mentioend above, MESSAGE also includes a detailed representation of carbon capture and sequestration from both fossil fuel and biomass combustion (see :numref:`tab_CCScapturerates`). 
+
+.. _tab_CCScapturerates:
+.. list-table:: Carbon capture rates in [%]
+   :widths: 25 45 15
+   :header-rows: 1
+
+   * - Conversion Process
+     - Plant type
+     - Capture rate
+   * - Electricity generation
+     - supercritical PC power plant with desulphurization/denox and CCS
+     - 90%
+   * - Electricity generation
+     - IGCC power plant with CCS
+     - 90%
+   * - Electricity generation
+     - biomass IGCC power plant with CCS
+     - 86%
+   * - Liquid fuel production
+     - Fischer-Tropsch coal-to-liquids with CCS
+     - 85%
+   * - Liquid fuel production
+     - coal methanol-to-gasoline with CCS
+     - 85%
+   * - Liquid fuel production
+     - Fischer-Tropsch gas-to-liquids with CCS
+     - 90%
```

Comment:
The capture rates for liquid fuel production from coal and gas seem very high to me given that a significant fraction of carbon remains in the liquid fuel. Are these relative to the carbon content of the input or relative to the output, excluding the carbon in the fuel?

Reply:
These are based on the output fuel.  I took these numbers from the spread sheet derived by "eric", where the capture efficiencies are explicitly provided. So, syn_liq has an entry into CO2c of .539.  syn_liq_ccs has an entry of 0.080 (therefore .459 is captured). The remainder .631 remains in the output fuel.

Reply:
Hi all,

Yes, this table also looks a bit unclear to me, especially regarding the liquid fuel production.

I'l probably drop by your office @OFR-IIASA to understand it a probably to make it clearer for an external reader. Afterwards I guess we could close and merge this PR.

Reply:
I had a closer look at this issue, using the example of coal methanol-to-gasoline with CCS (meth_coal_ccs in AFR for 2030). I can reproduce the 85% capture rate for the residual emissions, i.e. the emissions excluding the carbon that remains in the fuel produced. My suggestion would be to provide two capture rates with a clear definition in this table: (i) the current capture rate as a fraction of the process CO2 emissions and (ii) the capture rate relative to the carbon content of the feedstock. For the above example, the latter is about 55% as a significant share of carbon remains in the fuel. The former number is interesting from an engineering point of view and describes the efficiency of the capture process. The latter number is particularly relevant for the biomass-based CDR technologies as it describes the potential to sequester carbon from the atmosphere. Obviously for processes that produce carbon-free energy carriers like electricity of hydrogen the two are the same.
@OFR-IIASA Please add this second column with a clear definition. - Thanks.